thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/alert/content/EmailContentFormatterConfiguration.java

```diff
@@ -0,0 +1,41 @@
+package com.linkedin.thirdeye.alert.content;
+
+import com.linkedin.thirdeye.anomaly.ThirdEyeAnomalyConfiguration;
+import com.linkedin.thirdeye.common.ThirdEyeConfiguration;
+
+
+public class EmailContentFormatterConfiguration extends ThirdEyeConfiguration{
```

Comment:
Please do not inherit from ThirdEyeConfiguration, it's the config for a dropwizard application (webapp, controller). Instead just create a POJO with the required fields.

Reply:
thanks for the comment. It's been fixed.